scripts/build-data.js

```diff
@@ -118,31 +161,26 @@ const getLowestImplementedVersion = ({ features }, env) => {
 
   const envTests = tests
     .map(({ res: test, name, isBuiltIn }, i) => {
+
+      if (name === "const/basic support") {
+        console.log(test);
+      }
+
       // Babel itself doesn't implement the feature correctly,
       // don't count against it
       // only doing this for built-ins atm
       if (!test.babel && isBuiltIn) {
         return "-1";
       }
 
-      // `equals` in compat-table
-      Object.keys(test).forEach((t) => {
-        const invertedEnvs = invertedEqualsEnv[envMap[t] || t];
-        if (invertedEnvs) {
-          invertedEnvs.forEach((inv) => {
-            test[inv] = test[t];
-          });
-        }
-      });
-
       return Object.keys(test)
-      .filter((t) => t.startsWith(env))
-      // Babel assumes strict mode
-      .filter((test) => tests[i].res[test] === true || tests[i].res[test] === "strict")
-      // normalize some keys
-      .map((test) => envMap[test] || test)
-      .filter((test) => !isNaN(parseFloat(test.replace(env, ""))))
-      .shift();
+        .filter((t) => t.startsWith(env))
+        // Babel assumes strict mode
+        .filter((test) => tests[i].res[test] === true || tests[i].res[test] === "strict")
```

Comment:
We could merge 2 filters in 1 for perf, but it's just a nit and could be changed in separate PR aimed to optimize things like this.